.tributors

```diff
@@ -145,6 +152,13 @@
         "orcid": "0000-0001-9791-4404",
         "publish_email": true
     },
+    "po09i": {
+        "contributions": [
+            "doc"
+        ],
+        "email": "po09i@hotmail.com",
+        "name": "Alexandre D'Astous"
```

Comment:
part of git history here

```
❯ git log | grep po09i@hotmail.com | sort | uniq -c
     14 Author: Alexandre D'Astous <po09i@hotmail.com>
```

so public to that degree although not found by googling.
https://orcid.org/0000-0003-0381-7334

but I failed